src/Network/Network/AzureFirewall/ApplicationRule/NewAzureFirewallApplicationRuleCommand.cs

```diff
@@ -20,16 +20,12 @@
 
 namespace Microsoft.Azure.Commands.Network
 {
-    [Cmdlet(VerbsCommon.New, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "FirewallApplicationRule", SupportsShouldProcess = true, DefaultParameterSetName = AzureFirewallApplicationRuleParameterSets.TargetFqdn), OutputType(typeof(PSAzureFirewallApplicationRule))]
+    [Cmdlet(VerbsCommon.New, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "FirewallApplicationRule", SupportsShouldProcess = true, DefaultParameterSetName = AzureFirewallApplicationRuleParameterSets.Default), OutputType(typeof(PSAzureFirewallApplicationRule))]
     public class NewAzureFirewallApplicationRuleCommand : AzureFirewallBaseCmdlet
     {
+
         [Parameter(
             Mandatory = true,
-            ParameterSetName = AzureFirewallApplicationRuleParameterSets.TargetFqdn,
-            HelpMessage = "The name of the Application Rule")]
-        [Parameter(
-            Mandatory = true,
-            ParameterSetName = AzureFirewallApplicationRuleParameterSets.FqdnTag,
```

Comment:
This will cause `Name` is required to every parameter set. This is a breaking change to the parameter sets that does not need `Name` before.

Reply:
Name is a required parameter in the rule. If this was not the case, the earlier state was incorrect.

Reply:
https://docs.microsoft.com/en-us/powershell/module/azurerm.network/new-azurermfirewallapplicationrule?view=azurermps-6.13.0